### PR TITLE
Init sample for NodeJS

### PR DESCRIPTION
**Pull request by khanhdodang**

### CODE REVIEW

**Comment by khanhdodang** [comment]
cc @duylam @taynguyen Pls help me take a look this PR. Thanks

File: javascript/android-app-test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    catch (err) {
+      if (err.data) {
+        console.error(`init driver: ${err.data}`)
+      }
+    throw err
+    }
+  })
+
+  it('should show the app label', async () => {
+    await driver.init(desiredCaps)
```
Comment (taynguyen) [inline javascript/android-app-test.js]: driver was init twice? At before and in the test case.

File: javascript/ios-app-test.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import 'babel-polyfill'
+import 'colors'
+import wd from 'wd'
+import {assert} from 'chai'
+
+const username = 'nakitaxesi'
```
Comment (taynguyen) [inline javascript/ios-app-test.js]: Leaking username and apiKey.

File: javascript/ios-app-test.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    catch (err) {
+      if (err.data) {
+        console.error(`init driver: ${err.data}`)
+      }
+    throw err
+    }
+  })
+
+  it('should get text UIKitCatalog', async () => {
+    await driver.init(desiredCaps)
```
Comment (taynguyen) [inline javascript/ios-app-test.js]: driver was init twice.

File: javascript/ios-web-test.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import 'babel-polyfill'
+import 'colors'
+import wd from 'wd'
+import {assert} from 'chai'
+
+const username = 'nakitaxesi'
```
Comment (taynguyen) [inline javascript/ios-web-test.js]: Leaking username and apiKey

File: javascript/package.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "kobiton.test",
```
Comment (taynguyen) [inline javascript/package.json]: I think the name should be kobiton.sample.nodejs

**Comment by khanhdodang** [comment]
@taynguyen all comments are resolved. 

`username and apiKey were also leaking from gif image`: This is a fake apiKey. No worries about it.

Thanks em

File: javascript/README.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+- To update your npm, type this into your terminal:
+
+```bash
+  npm install npm@latest -g
+```
+
+
+## III. Installation
+
+- Clone [this script directly from our GitHub repo](https://github.com/kobiton/samples/tree/master/javascript) and run this command in your terminal:
```
Comment (duylam) [inline javascript/README.md]:
```
Clone [this script directly from our GitHub repo](https://github.com/kobiton/samples) and run this command in your terminal:
```

File: javascript/README.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+  
+  ```bash
```
Comment (duylam) [inline javascript/README.md]:
```
cd javascript
npm install
```

File: javascript/README.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+    npm install
+  ```
+
+
+## IV. Getting started with Kobiton
```
Comment (duylam) [inline javascript/README.md]: why not do similar with https://github.com/kobiton/samples/blob/master/java/README.md ?

File: javascript/README.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+  deviceGroup:        'KOBITON', 
+  deviceName:         'Galaxy',
+  platformName:       'Android'
+}
+
+```
+
+## V. Executing the Sample Tests
+
+### Once you have everything set up, you can run the example test simply by running the following command:
```
Comment (duylam) [inline javascript/README.md]: running **one of** following command:

**Pull request merged**